uaclient/api/u/pro/config/v1.py

```diff
@@ -101,7 +101,7 @@ class ConfigInfo(DataObject, AdditionalInfo):
                 " 'available'. If set to 'on', the guest will attach using the"
                 " host's Pro access automatically on launch. If set to 'off',"
                 " the guest will not be allowed to attach using the host's Pro"
-                " lx access. If set to 'available', the guest will be allowed"
```

Comment:
I think the more correct statement would be "...allowed to attach using the host's Pro access...". So I think we shuold remove "lx"/"lxd" entirely